src/mono/wasm/debugger/BrowserDebugProxy/MemberObjectsExplorer.cs

```diff
@@ -64,13 +64,9 @@ private static string GetNamePrefixForValues(string memberName, string typeName,
                 typePropertiesBrowsableInfo.TryGetValue(field.Name, out state);
             }
             fieldValue["__state"] = state?.ToString();
-
-            fieldValue["__section"] = field.Attributes switch
-            {
-                FieldAttributes.Private => "private",
-                FieldAttributes.Public => "result",
-                _ => "internal"
-            };
+            fieldValue["__section"] = field.Attributes.HasFlag(FieldAttributes.Private)
+                ? "private" : field.Attributes.HasFlag(FieldAttributes.Public)
```

Comment:
nit: I would prefer parentheses for the nested condition, to help with readability.